etc/firefox.profile

```diff
@@ -1,77 +1,74 @@
-# Persistent global definitions go here
+#Firejail profile for firefox
+#This file is overwritten after every install/update
+#Persistent global definitions
 include /etc/firejail/globals.local
-
-# This file is overwritten during software install.
-# Persistent customizations should go in a .local file.
+#Persistent local customizations
 include /etc/firejail/firefox.local
 
-# Firejail profile for Mozilla Firefox (Iceweasel in Debian)
-noblacklist ~/.mozilla
 noblacklist ~/.cache/mozilla
+noblacklist ~/.config/okularpartrc
+noblacklist ~/.config/okularrc
 noblacklist ~/.config/qpdfview
-noblacklist ~/.local/share/qpdfview
-noblacklist ~/.kde4/share/apps/okular
 noblacklist ~/.kde/share/apps/okular
+noblacklist ~/.kde4/share/apps/okular
 noblacklist ~/.local/share/okular
-noblacklist ~/.config/okularpartrc
-noblacklist ~/.config/okularrc
+noblacklist ~/.local/share/qpdfview
+noblacklist ~/.mozilla
 noblacklist ~/.pki
 
 include /etc/firejail/disable-common.inc
-include /etc/firejail/disable-programs.inc
 include /etc/firejail/disable-devel.inc
+include /etc/firejail/disable-programs.inc
 
-caps.drop all
-# ipc-namespace crashes firefox on some setups
-netfilter
-nogroups
-nonewprivs
-noroot
-protocol unix,inet,inet6,netlink
-seccomp
-shell none
-tracelog
-
-whitelist ${DOWNLOADS}
-mkdir ~/.mozilla
-whitelist ~/.mozilla
 mkdir ~/.cache/mozilla/firefox
+mkdir ~/.mozilla
+mkdir ~/.pki
+whitelist ${DOWNLOADS}
+whitelist ~/.cache/gnome-mplayer/plugin
 whitelist ~/.cache/mozilla/firefox
-whitelist ~/dwhelper
-whitelist ~/.zotero
-whitelist ~/.vimperatorrc
-whitelist ~/.vimperator
-whitelist ~/.pentadactylrc
-whitelist ~/.pentadactyl
-whitelist ~/.keysnail.js
 whitelist ~/.config/gnome-mplayer
-whitelist ~/.cache/gnome-mplayer/plugin
-mkdir ~/.pki
-whitelist ~/.pki
-whitelist ~/.lastpass
-whitelist ~/.config/qpdfview
-whitelist ~/.local/share/qpdfview
-whitelist ~/.config/okularrc
 whitelist ~/.config/okularpartrc
-whitelist ~/.kde4/share/apps/okular
+whitelist ~/.config/okularrc
+whitelist ~/.config/pipelight-silverlight5.1
+whitelist ~/.config/pipelight-widevine
+whitelist ~/.config/qpdfview
 whitelist ~/.kde/share/apps/okular
+whitelist ~/.kde4/share/apps/okular
+whitelist ~/.keysnail.js
+whitelist ~/.lastpass
 whitelist ~/.local/share/okular
-
-# silverlight
+whitelist ~/.local/share/qpdfview
+whitelist ~/.mozilla
+whitelist ~/.pentadactyl
+whitelist ~/.pentadactylrc
+whitelist ~/.pki
+whitelist ~/.vimperator
+whitelist ~/.vimperatorrc
 whitelist ~/.wine-pipelight
 whitelist ~/.wine-pipelight64
-whitelist ~/.config/pipelight-widevine
-whitelist ~/.config/pipelight-silverlight5.1
-
+whitelist ~/.zotero
+whitelist ~/dwhelper
 include /etc/firejail/whitelist-common.inc
 
-# experimental features
+caps.drop all
+netfilter
+nogroups
+nonewprivs
+noroot
+protocol unix,inet,inet6,netlink
+seccomp
+shell none
+tracelog
+
+# private-dev might prevent video calls going out
```

Comment:
@SpotComms Pedantic note here - I rather like having a space after the # in the comments (i.e. `# Firejail profile for XYZ` instead of `#Firejail profile for XYZ`). Thoughts?